Formula/gofabric8.rb

```diff
@@ -0,0 +1,36 @@
+class Gofabric8 < Formula
+  desc "CLI for fabric8 running on Kubernetes or OpenShift"
+  homepage "https://github.com/fabric8io/gofabric8/"
+  url "https://github.com/fabric8io/gofabric8/archive/v0.4.69.tar.gz"
+  sha256 "21a34018367d8aeb9446be23747ff918770d0c8e827e7d3e0b7a57542eb61274"
+
+  depends_on "go" => :build
+
+  GOVENDOR_PATH = "src/github.com/fabric8io/gofabric8".freeze
+
+  def install
+    ENV["GOPATH"] = buildpath
+    files = Dir["*"]
+    mkdir_p buildpath/GOVENDOR_PATH
+    mv files, buildpath/GOVENDOR_PATH
+    cd GOVENDOR_PATH
+    system "make", "install", "REV=homebrew"
```

Comment:
I think these fields are mainly useful for those who have built off a specific commit hash or their own branch  - given this is an officially cut release, it should be OK to take this approach, unless you object.